images/atomic-registry-quickstart/registry-console-template.yaml

```diff
@@ -0,0 +1,117 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: "registry-console-template"
+labels:
+  createdBy: "registry-console-template"
+parameters:
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: OPENSHIFT_OAUTH_PROVIDER_URL
+    required: true
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: COCKPIT_KUBE_URL
+    required: true
+  -
+    description: "The public url for the Openshift OAuth Provider"
+    name: COCKPIT_KUBE_INSECURE
+    required: false
+  -
+    description: "Oauth client secret"
+    name: OPENSHIFT_OAUTH_CLIENT_SECRET
+    from: "user[a-zA-Z0-9]{64}"
+    generate: expression
+  -
+    description: "Oauth client id"
+    name: OPENSHIFT_OAUTH_CLIENT_ID
+    value: "cockpit-oauth-client"
+  -
+    description: "Skip kubernetes CA verification"
+    name: KUBERNETES_INSECURE
+    value: ""
+  -
+    description: "PEM Encoded certificate to use for CA verification"
+    name: KUBERNETES_CA_DATA
+    value: ""
+objects:
+  -
+    kind: DeploymentConfig
+    apiVersion: v1
+    metadata:
+      name: "registry-console"
+    labels:
+      name: "registry-console"
+    spec:
+      replicas: 1
+      selector:
+      name: "registry-console"
+    template:
+      metadata:
+        labels:
+          name: "registry-console"
+      spec:
+        containers:
+          -
+            name: "registry-console"
+            image: "aweiteka/cockpit-registry:0.95"
+            ports:
+              -
+                containerPort: 9090
+                protocol: TCP
+            env:
+              -
+                name: OPENSHIFT_OAUTH_PROVIDER_URL
+                value: "${OPENSHIFT_OAUTH_PROVIDER_URL}"
+              -
+                name: OPENSHIFT_OAUTH_CLIENT_ID
+                value: "${OPENSHIFT_OAUTH_CLIENT_ID}"
+              -
+                name: KUBERNETES_INSECURE
+                value: "${KUBERNETES_INSECURE}"
+              -
+                name: KUBERNETES_CA_DATA
+                value: "${KUBERNETES_CA_DATA}"
+              -
+                name: COCKPIT_KUBE_INSECURE
+                value: "${COCKPIT_KUBE_INSECURE}"
+  -
+    kind: Service
+    apiVersion: v1
+    metadata:
+     name: "registry-console"
+     labels:
+       name: "registry-console"
+    spec:
+      type: ClusterIP
+    ports:
+      -
+        protocol: TCP
+        port: 9000
+        targetPort: 9090
+        selector:
+          name: "registry-console"
+  -
+    kind: ImageStream
+    apiVersion: v1
+    metadata:
+      name: registry-console
+      annotations:
+        description: Atomic Registry console
+    spec:
+      tags:
+        - annotations: null
+          from:
+            kind: DockerImage
+            name: aweiteka/cockpit-registry
+          name: "0.95"
+  -
+    kind: OAuthClient
+    apiVersion: v1
+    metadata:
+      name: "${OPENSHIFT_OAUTH_CLIENT_ID}"
+      respondWithChallenges: false
+    secret: "${OPENSHIFT_OAUTH_CLIENT_SECRET}"
```

Comment:
@smarterclayton this is a generated param. I don't think we need to use a secret vol.